.travis.yml

```diff
@@ -34,6 +34,7 @@ jobs:
     - chmod 600 ./ci/certs_key
     - eval "$(ssh-agent -s)"
     - ssh-add ./ci/certs_key
+    - echo "GOOGLE_PLACES_API_KEY=$GOOGLE_PLACES_API_KEY" > .env
```

Comment:
you might need to add this in above for the testing step

Reply:
i did thank u!